recipe/meta.yaml

```
@@ -31,9 +33,13 @@ build:
 
 requirements:
   build:
+    - git  # [not win]
+    # setuptools >= 65.0.0 is lacking the msvccompiler module in distutils.
+    # https://github.com/pypa/setuptools/pull/3505
+    - setuptools 63.4.1
```

```suggestion
    - setuptools <65.0.0
```
Less restrictive for maintenance. The same applies to all the usages below.